packages/grid/_modules_/grid/models/params/gridBaseComponentProps.ts

```diff
@@ -32,5 +32,5 @@ export interface GridBaseComponentProps {
   /**
    * The ref of the inner div Element of the grid.
    */
-  rootElement: GridRootContainerRef;
+  rootelement: GridRootContainerRef;
```

Comment:
Why this breaking change?

Reply:
There was a warning in the console because of the camelcase. I didn't see it being used anywhere but the changed places so I thought it is ok to change it.

Reply:
> There was a warning in the console because of the camelcas

Do you mean in your IDE or in the CI?

Reply:
I meant to say in the browser console.

Reply:
Oh, so there is a leak to the DOM we should fix. Please revert then. This warning was helpful.

Reply:
I've reverted it and added a potential solution to the problem. Let's move the discussion to the bottom thread to keep it in a single place. I'll resolve that one.